packages/core/metrics/core-metrics-server-internal/src/logging/get_ops_metrics_log.ts

```diff
@@ -55,6 +55,11 @@ export function getEcsOpsMetricsLog(metrics: OpsMetrics) {
       ).format('0.000')} }`
     : '';
 
+  const eventLoopUtilizationVal = process?.event_loop_utilization;
+  const eventLoopUtilizationMsg = eventLoopUtilizationVal
+    ? ` utilization: ${numeral(process?.event_loop_utilization.utilization).format('0.00000')}`
```

Comment:
Including active/idle here didn't seem useful enough to justify the increased message length. But it is easy to add if anyone feels otherwise.